documentation/modules/configuring/proc-generating-reassignment-json-files.adoc

```diff
@@ -2,40 +2,177 @@
+
+Secure the reassignment process using TLS. You'll need a Kafka cluster that uses TLS encryption and authentication.
 
 .Prerequisites
 
-* A running Cluster Operator
-* A `Kafka` resource
-* A set of topics to reassign the partitions of
+* You have a running Cluster Operator.
+* You have a running Kafka cluster based on a `Kafka` resource configured with internal TLS authentication and encryption.
++
+.Kafka configuration with TLS
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    # ...
+    listeners:
+      # ...
+      - name: tls
+        port: 9093
+        type: internal
+        tls: true <1>
+        authentication:
+          type: tls <2>
+    # ...
+----
+<1> Enables TLS encryption for the internal listener.
+<2> Listener authentication mechanism xref:assembly-securing-kafka-brokers-str[specified as mutual TLS].
+* The running Kafka cluster contains a set of topics and partitions to reassign.
++
+.Example topic configuration for `my-topic`
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaTopicApiVersion}
+kind: KafkaTopic
+metadata:
+  name: my-topic
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  partitions: 10
+  replicas: 3
+  config:
+    retention.ms: 7200000
+    segment.bytes: 1073741824
+    # ...
+----
+* You have a `KafkaUser` configured with ACL rules that specify permission to produce and consume topics from the Kafka brokers.
++
+--
+.Example Kafka user configuration with ACL rules for producing and consuming from `my-topic`
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaUserApiVersion}
+kind: KafkaUser
+metadata:
+  name: my-user
+  labels:
+    strimzi.io/cluster: my-cluster
+spec:
+  authentication: <1>
+    type: tls
+  authorization:
+    type: simple <2>
+    acls:
+      - resource:
+          type: topic
+          name: my-topic
+          patternType: literal
+        operation: Write
+        host: "*"
+      - resource:
+          type: topic
+          name: my-topic
+          patternType: literal
+        operation: Create
+        host: "*"
+      - resource:
+          type: topic
+          name: my-topic
+          patternType: literal
+        operation: Describe
+        host: "*"
+  # ...
+----
+<1> User authentication mechanism defined as mutual `tls`.
+<2> Simple authorization and accompanying list of ACL rules.
+
+NOTE: Permission for a `Describe` operation is required as a minimum for TLS access to a topic.
+--
 
 .Procedure
 
-. Prepare a JSON file named `_topics.json_` that lists the topics to move.
-It must have the following structure:
+. Extract the cluster CA certificate and password from the `_CLUSTER-NAME_-cluster-ca-cert` Secret of the Kafka cluster.
 +
-[source,subs=+quotes]
+[source,shell,subs="+quotes"]
+kubectl get secret _CLUSTER-NAME_-cluster-ca-cert -o jsonpath='{.data.ca\.p12}' | base64 -d > ca.p12
```

Comment:
@ShubhamRwt  - I used the same jsonpath format we use in other parts of the guide for the same command. That is, no `"` or `[`. as with `"jsonpath={.data['user\.p12']}"`